sdk/src/backend/dlt.rs

```diff
@@ -163,15 +158,15 @@ mod tests {
             .match_header(HEADER_X_APP_NAME, AUTH_PROVIDER)
             .match_header("authorization", format!("Bearer {}", TOKEN.as_str()).as_str())
             .match_header("content-type", "application/json")
-            .match_query(Matcher::Exact(format!("network_id={}", iota_network_id)))
+            .match_query(Matcher::Exact(format!("network_key={}", iota_network_key)))
             .match_body(Matcher::Exact(body))
             .with_status(status_code)
             .expect(1)
             .with_header("content-type", "application/json")
             .create();
 
         // Act
-        let response = put_user_address(&config, &TOKEN, String::from("67a1f08edf55756bae21e7eb"), ADDRESS).await;
+        let response = put_user_address(&config, &TOKEN, String::from("IOTA"), ADDRESS).await;
```

Comment:
we can use the variable here:  

```suggestion
        let response = put_user_address(&config, &TOKEN, iota_network_key.to_string(), ADDRESS).await;
```